src/MartinCostello.BrowserStack.Automate/Build.cs

```diff
@@ -26,7 +26,7 @@ public class Build
         /// <summary>
         /// Gets or sets the duration of the build, in seconds.
         /// </summary>
-        [JsonProperty("duration")]
+        [JsonProperty("duration", NullValueHandling = NullValueHandling.Ignore)]
```

Comment:
This was coming back as null for a few builds when running against JE's BrowserStack account.